client/src/components/BuildingStatsTable.js

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+
+import Helpers from "util/helpers";
+
+import { withI18n } from "@lingui/react";
+import { t } from "@lingui/macro";
+import { Trans } from "@lingui/macro";
+
+const BuildingStatsTableWithoutI18n = (props) => {
+  const { i18n } = props;
+  const { boro, block, lot } = Helpers.splitBBL(props.addr.bbl);
+
+  const bblDash = (
+    <span className="unselectable" unselectable="on">
+      -
+    </span>
+  );
+
+  const BBL = () => (
+    <div
+      className="double"
+      title={i18n._(
+        t`This is the official identifer for the building according to the Dept. of Finance tax records.`
+      )}
+    >
+      <label>
+        Boro{bblDash}Block{bblDash}Lot (BBL)
+      </label>
+      {boro}
+      {bblDash}
+      {block}
+      {bblDash}
+      {lot}
+    </div>
+  );
+
+  const YearBuilt = () => (
+    <div
+      title={i18n._(
+        t`The year that this building was originally constructed, according to the Dept. of City Planning.`
+      )}
+    >
+      <Trans render="label">Year Built</Trans>
+      {props.addr.yearbuilt !== 0 ? props.addr.yearbuilt : "N/A"}
+    </div>
+  );
```

Comment:
@toolness something about defining these components as constants within the larger component felt like I was doing something very very wrong! At the same time, this felt like the DRYest solutions as it meant I didn't need to pass down properties to the smaller components at al. Let me know if this is kosher!